proxy/proxy.go

```diff
@@ -79,14 +87,24 @@ type Validator func(path string) bool
 // Factory returns a new Proxy instance.
 type Factory func() Proxy
 
-// GetProxy returns an instance of the correct proxy for the project
+// Get returns an instance of the correct proxy for the project
 //rooted at the given path.
-func GetProxy(path string) (Proxy, error) {
-	if java.IsValid(path) {
-		return java.Factory(), nil
-	}
-	if golang.IsValid(path) {
-		return golang.Factory(), nil
+func Get(path string) (Proxy, error) {
```

Comment:
I renamed this function so it doesn't stutter when invoked.

`proxy.GetProxy("/tmp")` => `proxy.Get("/tmp")`